index.html

```diff
@@ -15,6 +15,11 @@
         <div id="articles">Articles</div>
         <aside id="side-menu">Side menu</aside>
     </main>
-    <footer>Footer</footer>
+    <footer>
+        <table style="width:100%">
+            <td><a href="#">ABOUT ME</a></td>
```

Comment:
Nested tag in html will increase readability, try to fix all in file
```suggestion
            <td>
                    <a href="#">ABOUT ME</a>
            </td>
```